index.html

```diff
@@ -330,12 +330,21 @@ <h2>Introduction</h2>
 controller of an identifier, including material for cryptographic proofs and
 service endpoints for additional communications.
       </p>
-
       <p>
-It is expected that other specifications, such as [[[?DID-CORE]]], will profile
+A [=controller document=] specifies one or more
+[=verification relationships=] and [=service endpoints=] for a single 
+identifier, for which the current controller document is taken as authoritative. 
+Every controller document is stored and retrieved according to the URL of the 
+document, which is also the canonical identifier of the document.
```

Comment:
```suggestion
identifier, about which relationships the current controller document is taken
as authoritative. Every controller document is stored and retrieved according to
the URL of the document, which is also the canonical identifier of the document.
```

Reply:
It not just the relationships.  The entire document is authoritative or not.